src/automations/Job.ts

```diff
@@ -292,10 +294,15 @@ export default class Job {
                 return jobInfo;
             },
             {
-                htmlAsStr: JSON.parse(decodeURIComponent(innerHTML))["html"],
+                htmlAsStr: content["html"],
                 recruiterTag: RECRUITER,
             }
         );
+
+        return {
+            jobs,
+            hasMorePage: !!content["pagination"]?.replace(/\n/g, ""),
```

Comment:
Could we do this more explicit? Maybe with a ternary operator... I don't find it obvious when this is going to be `true` or `false`

Reply:
Do you mean something like ```content["pagination"]?.replace(/\n/g, "") ? true : false ``` or 
```content["pagination"] ? (content["pagination"].replace(/\n/g, "") ? true : false) : false ```

Reply:
I'm most likely missunderstanding this but... what could be `content["pagination"] for `hasMorePage` to be true?

Reply:
If there is some HTML it means there are more page if its "\n" then there is no more page

Reply:
Why not just `hasMorePage: content.pagination !== '\n'`?

Reply:
Yes i think it would be more readable